scripts/zones/Batallia_Downs/mobs/Skirling_Liger.lua

```diff
@@ -0,0 +1,37 @@
+-----------------------------------
+-- Area: Batallia Downs (105)
+--  MOB: Skirling Liger
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+end;
+
+-----------------------------------
+-- onMobEngaged
+-----------------------------------
+
+function onMobEngaged(mob,target)
+    mob:setMod(MOD_REGAIN, 50);
+end;
+
+-----------------------------------
+-- onMobDisengage
+-----------------------------------
+
+function onMobDisengage(mob)
+    mob:setMod(MOD_REGAIN,0);
+end;
+
+-----------------------------------
+-- onMobDeath
+-----------------------------------
+
+function onMobDeath(mob, player, isKiller)
+    UpdateNMSpawnPoint(mob:getID());
```

Comment:
or onmobdespawn?

On Tue, Jun 7, 2016 at 7:29 PM, Tahir Akhlaq <notifications@github.com>
wrote:

> In scripts/zones/Batallia_Downs/mobs/Skirling_Liger.lua
> <https://github.com/DarkstarProject/darkstar/pull/3218#discussion_r66180893>
> :
>
> > +end;
> > +
> > +-----------------------------------
> > +-- onMobDisengage
> > +-----------------------------------
> > +
> > +function onMobDisengage(mob)
> > +    mob:setMod(MOD_REGAIN,0);
> > +end;
> > +
> > +-----------------------------------
> > +-- onMobDeath
> > +-----------------------------------
> > +
> > +function onMobDeath(mob, player, isKiller)
> > +    UpdateNMSpawnPoint(mob:getID());
>
> same here
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DarkstarProject/darkstar/pull/3218/files/bc99a64acca7157c4184d8c3d594a9b2d7b6e59b#r66180893>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABGI_0bHLSvICiGWdpJBQImzYPSPWTciks5qJhsNgaJpZM4IwKEH>
> .
>